components/builder-api-client/src/lib.rs

```diff
@@ -199,22 +201,84 @@ pub struct OriginChannelIdent {
 
 #[derive(Clone, Deserialize)]
 pub struct OriginInvitation {
-    pub id:           String,
-    pub account_id:   String,
+    #[serde(with = "json_u64")]
+    pub id: u64,
+    #[serde(with = "json_u64")]
+    pub account_id: u64,
     pub account_name: String,
-    pub created_at:   String,
-    pub ignored:      bool,
-    pub origin:       String,
-    pub owner_id:     String,
-    pub updated_at:   String,
+    pub created_at: String,
+    pub ignored: bool,
+    pub origin: String,
+    #[serde(with = "json_u64")]
+    pub owner_id: u64,
+    pub updated_at: String,
 }
 
+#[derive(Clone, Deserialize)]
+pub struct UserOriginInvitationsResponse(pub Vec<OriginInvitation>);
+
 #[derive(Clone, Deserialize)]
 pub struct PendingOriginInvitationsResponse {
     pub origin:      String,
     pub invitations: Vec<OriginInvitation>,
 }
 
+macro_rules! invite_format {
```

Comment:
Why is this a macro? Seems like it could just be a straightforward constant.

Reply:
I can re-visit that strategy. When I first tried with a constant I encountered some issues.

Reply:
This is no longer relevant since I've removed the macro and introduced a Tabular trait in https://github.com/habitat-sh/habitat/pull/7349/commits/a65aebfafec7355e188e527768485ba9957fa404